prototype_source/fx_graph_mode_ptq_static_tutorial.py

```diff
@@ -0,0 +1,446 @@
+"""
+(prototype) FX Graph Mode Post Training Static Quantization
+===========================================================
+
+**Author**: `Jerry Zhang <https://github.com/jerryzh168>`_
+
+This tutorial introduces the steps to do post training static quantization in graph mode based on
+`torch.fx <https://github.com/pytorch/pytorch/blob/master/torch/fx/__init__.py>`_. 
+The advantage of FX graph mode quantization is that as long as the model can be symbolically traced, 
+we can perform quantization fully automatically on the model. 
+
+We have a separate tutorial for FX Graph Mode Post Training Dynamic Quantization(todo).	
+
+tldr; The FX Graph Mode API looks like the following:
+
+.. code:: python
+
+    import torch
+    from torch.quantization import get_default_qconfig
+    # Note that this is temporary, we'll expose these functions to torch.quantization after official releasee
```

Comment:
The wording "after official release" is ambiguous here.  Suggestion to rephrase - "Note that this is temporary and we will expose these functions to torch.quantization after this feature is no longer in prototype and moved to stable"

Reply:
oh, this will change before the prototype release. I'll have a PR later to fix this

Reply:
I'm guessing this isn't implemented yet. If you would like this to merge without building the file, you can rename the tutorial, removing _tutorial from the name. The tutorial will be available and you can even open it in colab, but the page will not be rendered with output during the build.

Reply:
this is implemented, it should work in master. is it possible for this to be build with master?

Reply:
We built tutorials with stable, so if it's not there it won't build. Should it work in 1.7.1 now?

Reply:
I see, it probably won't work in 1.7.1, then do people publish tutorials after the stable release?
I think we can try to merge without building the file then. I'll remove _tutorial from the file.

Reply:
Yes then we can publish the recipe to the regular location instead of prototype. :)

Reply:
oh, prototype means the feature introduced by the tutorial is a prototype feature. v.s. beta/stable. The tutorial will be marked as prototype in 1.8, I'm not sure if we can publish it in regular location or not.